client/ayon_slack/plugins/publish/integrate_slack_api.py

```diff
@@ -31,11 +34,16 @@ def process(self, instance):
         review_path = self._get_review_path(instance)
 
         publish_files = set()
+        message = ''
```

Comment:
Please use double quotes over single quotes if possible.
```suggestion
        message = ""
```